conda_smithy/cli.py

```diff
@@ -68,8 +95,30 @@ def __call__(self, args):
         feedstock_directory = args.feedstock_directory.format(package=argparse.Namespace(name=meta.name()))
         generate_feedstock_content(feedstock_directory, args.recipe_directory)
         if not args.no_git_repo:
-            create_git_repo(feedstock_directory, meta)
+            msg = 'Initial commit of the {} feedstock.'.format(meta.name())
+            create_git_repo(feedstock_directory, msg)
 
+class InitPile(Subcommand):
```

Comment:
Another idea would be to add a "--multi" switch to init and branch out depending if this switch is given or not...